src/modules/new-request-form/usePrefilledTicketFields.tsx

```diff
@@ -102,6 +117,12 @@ function getPrefilledTicketFields(fields: Fields): Fields {
               : "";
         }
         break;
+      case "due_at":
+      case "date":
+        if (isValidDate(sanitizedValue)) {
```

Comment:
If it is not valid we just won't prefill the field?

Reply:
What else can we do, a part eventually showing a message in the console?